client/apps/Admin/src/components/BulkAssignmentForm.tsx

```
@@ -0,0 +1,263 @@
+     * @description Confirm and execute bulk creation
+     */
+    function confirmBulkCreate(): void {
+        bulkCreateMutation.mutate({
+            userIds: selectedUserIds,
+            taskIds: selectedTaskIds,
+        });
+    }
+
+    return (
+        <div className="modal-overlay" onClick={onClose}>
+            <div className="modal bulk-assignment-form" onClick={(e) => e.stopPropagation()}>
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)